sandbox/react-vite-default-ts/src/stories/useExampleHook.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import useExampleHook from './useExampleHook';
+
+const meta: Meta<typeof useExampleHook> = {
+    title: 'Example/useExampleHook',
+    component: useExampleHook,
+    tags: ['autodocs'],
+    parameters: {
+        // Layout configuration for the story
+        layout: 'centered',
+    },
+} satisfies Meta<typeof useExampleHook>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+/**
+ * Default example story for `useExampleHook`.
+ *
+ * This story demonstrates the usage of the `useExampleHook` by
+ * displaying the returned message from the hook.
+ */
+export const Default: Story = {
```

Comment:
If there is no message returned from the hook, how would the Story look like? Or should there always be a message? Also would be nice to see some unit tests, even if the functionality seems pretty straightforwrd